src/main/java/io/vertx/redis/client/impl/RedisConnectionInternal.java

```diff
@@ -27,4 +30,14 @@ public interface RedisConnectionInternal extends RedisConnection {
    * Returns {@code true} is this connection can be reset. This means that the connection didn't enter PubSub mode.
    */
   boolean reset();
+
+  boolean isTainted();
```

Comment:
javadoc :-)

Reply:
thanks, I know it was there before, but I think it is important to define what it means clearly